packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -94,7 +94,7 @@ def test_if_marathon_pods_can_be_deployed_with_mesos_containerizer(dcos_api_sess
                 'name': 'ct1',
                 'resources': {'cpus': 0.1, 'mem': 32},
                 'image': {'kind': 'DOCKER', 'id': 'debian:jessie'},
-                'exec': {'command': {'shell': 'touch foo'}},
+                'exec': {'command': {'shell': 'touch foo; while true; do sleep 1; done'}},
```

Comment:
Because marathon is made to support long-running processes. This seems inherent to me, but more comments the merrier I guess